File: modul4/C/PRAK403-2310817210023-RifkyPutraMahardika.c
#include <stdio.h>
int main(){
    int angka1, angka2;
    printf("Input: ");
    scanf("%d %d", &angka1, &angka2);

    printf("Output: ");
    if(angka1<angka2){
        for(int i=angka1; i<=angka2; i++) {
            printf("%d %d", i, angka1+angka2-i);
            if (i<angka2) {
                printf(" - ");
            }
        }
    }else if(angka2<angka1){
        for(int i=angka1; i>=angka2; i--){
            printf("%d %d", i, angka1+angka2-i);
           if (i>angka2) {
                printf(" - ");
            }
        }
    }

    return 0;
}

File: modul4/Python/PRAK403-2310817210023-RifkyPutraMahardika.py
angka1, angka2 = map(int, input("Input: "). split())

print("Output: ", end="")
if angka1 < angka2:
    for i in range(angka1, angka2+1):
        print(i, angka1+angka2-i, end="")
        if i < angka2:
            print(" - ", end="")
elif angka2 < angka1:
    for i in range(angka1, angka2-1, -1):
        print(i, angka1+angka2-i, end="")
        if i > angka2:
            print(" - ", end="")